src/index.html

```diff
@@ -7,28 +7,59 @@ <h1>Instant, themed Twitch chat overlays</h1>
 	<form id="overlay-builder">
 		<label for="channel">Channel name</label>
 		<input type="text" id="channel" name="channel" required="true" autocomplete="off" autocorrect="off"
-			autocapitalize="off" spellcheck="false" />
-		<label for="theme">Theme</label>
+			autocapitalize="off" spellcheck="false" placeholder="{{defaults.channel}}"
+			title="Your Twitch channel name" />
+
+		<div class="overlay-builder-header">
+			<label for="theme">Theme</label>
+			<a href="docs/configuration/#theme" aria-label="Docs for theme">&#9432;</a>
+		</div>
 		<select id="theme" name="theme">
-			<option value="default">Default</option>
+			<option value="{{defaults.theme}}">{{defaults.theme | formatThemeName}}</option>
```

Comment:
I think the name "Default" here is independent of the underlying value `default`. Would you be open to hardcoding the name "Default"?

```suggestion
			<option value="{{defaults.theme}}">Default</option>
```